lahja/endpoint.py

```diff
@@ -185,7 +312,11 @@ def broadcast(self, item: BaseEvent, config: Optional[BroadcastConfig] = None) -
             # and are directly put into the local receiving queue instead.
             self._internal_queue.put_nowait((item, config))
         else:
-            self._sending_queue.put_nowait((item, config))
+            # Broadcast to every connected Endpoint that is allowed to receive the event
+            for name, connector in self._connected_endpoints.items():
+                verboten = config is not None and not config.allowed_to_receive(name)
```

Comment:
nitpick: as much as I love German (:smile:) the double-negative is a little hard for me to read. Something like:

```python
allowed = (config is None) or config.allowed_to_receive(name)
if allowed:
   ...
```

might be easier to read.

Reply:
Yes, that looks better. :+1:  Are the parenthesis only for better logical grouping or are they actually needed?